File: java/src/main/java/pizza/repository/TimeRepository.java
package pizza.repository;


import org.springframework.jdbc.core.JdbcTemplate;
import org.springframework.stereotype.Repository;

@Repository
public class TimeRepository {
    private JdbcTemplate jdbcTemplate;

    public TimeRepository(JdbcTemplate jdbcTemplate) {
        this.jdbcTemplate = jdbcTemplate;
    }

    public String getTime() {
        return jdbcTemplate.queryForObject("select current_timestamp", String.class);
    }
}

File: kotlin-mvc/src/main/kotlin/com/hyvee/application/repositories/TimeRepository.kt
package com.hyvee.application.repositories

import org.springframework.jdbc.core.JdbcTemplate
import org.springframework.stereotype.Repository

@Repository
class TimeRepository(private val jdbcTemplate: JdbcTemplate) {
    fun getTime(): String? {
        return jdbcTemplate.queryForObject("select current_timestamp", String::class.java)
    }
}
